tests/clients/detectionlists/test_detection_list_user_client.py

```diff
@@ -1,11 +1,20 @@
 import json
```

Comment:
File `detection_list_user_client` was moved to `_internal.clients` but the respective test file wasn't while fixing docs issue in HRE.

Reply:
thanks for catching that!

Reply:
moved test file to appropriate location.